test/integration/gardener/scheduler/scheduler_test.go

```diff
@@ -177,8 +179,7 @@ var _ = Describe("Scheduler testing", func() {
 
 		if testMachineryRun != nil && *testMachineryRun {
 			schedulerOperationsTestLogger.Info("Running in test Machinery")
-			zero := int32(0)
-			replicas, err := ScaleGardenerControllerManager(setupContextTimeout, schedulerGardenerTest.ShootGardenerTest.GardenClient.Client(), &zero)
+			replicas, err := ScaleGardenerControllerManager(setupContextTimeout, schedulerGardenerTest.ShootGardenerTest.GardenClient.Client(), pointer.Int32Ptr(0))
```

Comment:
Hmm, this is still from "old times", where we didn't have the gardenlet, right?
@schrodit How/against which landscape is this run again? Or was it even disabled? I have already asked this question, but I can't remember...

Reply:
Valid question, though unrelated to this PR as it is updating the Maintenace integration tests and not the scheduler tests.

Reply:
currently theses tests do not run. As you pointed out, they could be disruptive when scheduling doen necessary components (scheduler) or modifying the cloudprofile.

Therefore we do not want to run them on a productive landscape (also staging or dev does not seem right).
These tests can be executed as soon as we are able to easily spin up gardener clusters and test them in isolation.

So for me this PR is fine. And we will run them in the testmachinary as soon as we are able to create isolated landscapes.

Reply:
Alright, thanks for clarifying!